spylon_kernel/init_spark_magic.py

```diff
@@ -48,16 +49,19 @@ def get_completions(self, info):
             return []
 
         text = info['code']
-        interpreter = Interpreter(text, [self.env])
-
         position = (info['line_num'], info['column'])
-        path = UserContext(text, position).get_path_until_cursor()
-        path, dot, like = completion_parts(path)
-        before = text[:len(text) - len(like)]
+        interpreter = jedi.Interpreter(text, [self.env])
```

Comment:
if jedi is not available in the environment, isn't this going to raise a Type error? Something along the lines of `None type has no attribute Interpreter`. Perhaps this needs a `if jedi is not None` somewhere in here?